CHANGELOG.md

```diff
@@ -12,6 +12,8 @@
 
 * **Fixes a metadata source serialization bug** Problem: In unstructured elements, when loading an elements json file from the disk, the data_source attribute is assumed to be an instance of DataSourceMetadata and the code acts based on that. However the loader did not satisfy the assumption, and loaded it as a dict instead, causing an error. Fix: Added necessary code block to initialize a DataSourceMetadata object, also refactored DataSourceMetadata.from_dict() method to remove redundant code. Importance: Crucial to be able to load elements (which have data_source fields) from json files.
 
+* **Fixes issue when channel raises `None`** Retrieves the channel only once, moves `jump_url` property setting within the message retrieval context.
+
```

Comment:
follow structure of the log above this one:
title should clearly state that this fixes issue where channel isn't found by the Discord source connector.
then state the Problem: describe what is currently broken and any relevant background. Then state the Fix: what changed in order to fix this. Finally include the Importance: how this impacts the user.